cirq_superstaq/ops/qubit_gates_test.py

```diff
@@ -218,9 +249,12 @@ def test_acecr_qasm() -> None:
         qreg q[2];
 
 
-        acecr_pm q[0],q[1];
-        acecr_mp q[1],q[0];
-        acecr_mp_rx(pi*0.5) q[0],q[1];
+        acecr q[0],q[1];
+        acecr_rzx(pi*-0.5) q[1],q[0];
+        acecr_rx(pi*0.5) q[0],q[1];
+        acecr_rzx_rx(pi*-0.5,pi*0.5) q[0],q[1];
+        acecr_rzx(pi*0.3333333333) q[0],q[1];
+        acecr_rzx_rx(pi*0.3333333333,pi*0.5) q[0],q[1];
```

Comment:
(assuming prior suggestion)
```suggestion
        acecr_rzx(pi*0.2) q[0],q[1];
        acecr_rzx_rx(pi*0.2,pi*0.5) q[0],q[1];
```